### PR TITLE
Form styles

### PR DESCRIPTION
**Pull request by adamccfc**

### CODE REVIEW

File: app/assets/stylesheets/govwales/components/_form.scss [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      color: $white;
+      font-weight: bold;
+      font-size: 20px;
+      font-size: 1.25rem;
+      padding: 15px 20px;
+      margin-bottom: 0;
+      box-shadow: none;
+
+      &:hover {
+        background-color: $brightred;
```
Comment (markheathy) [inline app/assets/stylesheets/govwales/components/_form.scss]: Adjust this to a govwales var

File: app/assets/stylesheets/govwales/components/_form.scss [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    .govuk-form-group {
+      &--error {
+        padding-left: 0;
+        border-left: none;
+      }
+    }
+
+    // Submit button
+    .govuk-button {
+      background-color: $red;
```
Comment (markheathy) [inline app/assets/stylesheets/govwales/components/_form.scss]: Adjust this to a govwales var

File: app/assets/stylesheets/govwales/components/_form.scss [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+
+      &:active {
+        top: 0;
+      }
+    }
+  }
+
+  .govuk-error-summary,
+  .govuk-error-message {
+    color: $form__text;
```
Comment (markheathy) [inline app/assets/stylesheets/govwales/components/_form.scss:123]: At some stage, maybe set up a sub-task to sweep the vars and prefix 'govwales_' to them? Just incase we duplicate alphagov variables? What do you think? Maybe not that much of a big deal if we are the last applied CSS in the chain but helps with handling. Phase 2.

**Review by markheathy — COMMENTED** [review]
Review done @adamccfc. Thanks.

File: app/assets/stylesheets/govwales/components/_form.scss (hunk 3 above)
Comment (adamccfc) [reply app/assets/stylesheets/govwales/components/_form.scss:123]: @markheathy for speed I don't think we need to worry about this at the moment. Might be nice to have going forward but as long as the styles are being applied then I don't think we should worry about it too much at the moment.

File: app/assets/stylesheets/govwales/components/_form.scss (hunk 1 above)
Comment (adamccfc) [reply app/assets/stylesheets/govwales/components/_form.scss]: I've changed this to a pre existing form var now

File: app/assets/stylesheets/govwales/components/_form.scss (hunk 2 above)
Comment (adamccfc) [reply app/assets/stylesheets/govwales/components/_form.scss]: I've changed this to a pre existing form var now

File: app/assets/stylesheets/govwales/components/_form.scss (hunk 3 above)
Comment (markheathy) [reply app/assets/stylesheets/govwales/components/_form.scss:123]: Happy to go with that.

**Pull request merged**